### DISCUSSION TITLE
error=HTTP status client error (400 Bad Request) for url (https://api.aliyundrive.com/v2/file/list)

### DISCUSSION THREAD

**Discussion by duaka** [messense/aliyundrive-webdav#657] [Q&A]
### 问题描述

run aliyundrive-webdav in WSL1 ubuntu, windows 10 LTSC 21H2 map network drive. When play media file, report 

```
aliyundrive_webdav::vfs: list_files_and_cache failed path=....../file.mp4 error=HTTP status client error (400 Bad Request) for url (https://api.aliyundrive.com/v2/file/list)
```

### 重现步骤

always

### 版本

1.10.1

### 运行平台

python venv on WSL1 Ubuntu

### refresh token 获取方式

网页版

### 日志

2022-09-28T00:36:17.599027Z ERROR aliyundrive_webdav::vfs: list_files_and_cache failed path=<file>.mp4 error=HTTP status client error (400 Bad Request) for url (https://api.aliyundrive.com/v2/file/list)

**Comment by wuxiaoqiang12** [comment]
I have this issue too. Do you guys have any suggestions?
